src/InputField.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { Input, Label, Text, Flex, Box, Icon } from '../src'
+
+const InfoBox = styled(Box)`
+  border-width: 1px;
+  border-radius: 0 0 2px 2px;
+  border-color: ${({color}) => color};
+  border-style: solid;
+  box-shadow: 0 -1px 0 0 ${({color}) => color};
+`
+const propTypes = {
+  /** Id passed to the native input element. Necessary to set up aria-describedby and for attributes for accessibility.  */
+  id: PropTypes.string.isRequired,
+  /** Text to be used as the label for the input element. */
+  label: PropTypes.string.isRequired,
+  /** The optional icon that shows up on the right side of the element. */
+  icon: PropTypes.string,
+  /** Optional border and icon color. */
+  color: PropTypes.string,
+  /** Additional content to go in the info drawer below the element. This will be replaced by a tooltip later probably.  */
+  info: PropTypes.string
+}
+const InputField = ({
+  id,
+  label,
+  color,
+  icon,
+  info,
+  ...otherProps
```

Comment:
I feel like the typical convention for naming this is either `...props` or `...rest` - maybe consider sticking to a common convention?